MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,237 @@
+
+{% tabs %}
+{% highlight C# %}
+
+    DataFormGroupItem groupItem = new DataFormGroupItem();
+        groupItem.Name = "Address";
+        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "Address" });
+        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "State" });
+        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "ZipCode" });
+        this.dataForm.Items.Add(groupItem);
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight C# %}
+
+    public class DataFormModel
+    {
+        public string Name { get; set; }
+        public int Phone { get; set; }
+    }
+
+
+    public class DataFormViewModel
+    {
+        public DataFormViewModel
+        {
+            this.DataFormModel = new DataFormModel();
+        }
+
+        public DataFormModel DataFormModel { get; set; }
+        
+    }
+{% endhighlight %}
+{% endtabs %}
+
+`AddCustomItemsimages`
+
+### Dynamically remove manually added dataform items
+You can dynamically remove the dataform items from collections using the `Items` property of `SfDataForm`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.Items.RemoveAt(2);
+
+{% endhighlight %}
+{% endtabs %}
+
+
+### Dynamically clear manually added dataform items
+You can dynamically clear the dataform items using the `Items` property of `SfDataForm`.
+
+{% tabs %}
+{% highlight C# %}
+
+    dataForm.Items.Clear();
+
+{% endhighlight %}
+{% endtabs %}
+
+`images/ClearCustomItems`
+
+### Dynamically reset manual dataform items
+You can reset the dataform items using the `Items` property of `SfDataForm` and we can reset two or more dataform items at a time.
+
+{% tabs %}
+{% highlight C# %}
+
+    DataFormTextItem textItem = new DataFormTextItem() { FieldName = "Age" };
+    dataForm.Items[2] = textItem;
+    
+    
+    DataFormDateItem dateItem = new DataFormDateItem() { FieldName ="Date Of Birth", RowOrder = 2, ItemsOrderInRow = 1, ColumnSpan = 3 });
+
+    dataForm.Items[3] = dateItem;
+    
+{% endhighlight %}
+{% endtabs %}
+
+
+### Dynamically add Dataform group items
+You can dynamically add custom group items using `Items` property of `SfDataForm`.
+{% tabs %}
+{% highlight C# %}
+
+    DataFormGroupItem dataFormGroupItem = new DataFormGroupItem();
+    dataFormGroupItem.Name = "GroupItem";
+    dataFormGroupItem.IsExpanded = true;
+    dataFormGroupItem.Items = new ObservableCollection<DataFormItem>();
+
+    dataFormGroupItem.Items.Add(new DataFormTextItem() { FieldName = "First Name", ItemsOrderInRow = 0 , GroupName = "GroupItem" });
+    dataFormGroupItem.Items.Add(new DataFormTextItem() { FieldName = "Middle Name", ItemsOrderInRow = 1 , GroupName = "GroupItem" });
+    dataFormGroupItem.Items.Add(new DataFormTextItem() { FieldName = "Last Name", ItemsOrderInRow = 2 , GroupName = "GroupItem" });
+
+    dataForm.Items.Add(dataFormGroupItem);
+{% endhighlight %}
+{% endtabs %}
+
+`images/GroupItems`
+
+#### Adding custom editor view using Manual DataFormItem
```

Comment:
You can directly set a custom view as an `EditorView` of `dataForm:DataFormCustomItem`. In this case, the editor view will not be generated based on field type and `RegistorEditor`.

Reply:
content added